modules/network-observability-SRIOV-configuration.adoc

```diff
@@ -0,0 +1,36 @@
+// Module included in the following assemblies:
+//
+// network_observability/configuring-operator.adoc
+
+:_content-type: PROCEDURE
+[id="network-observability-SR-IOV-config_{context}"]
+= Configuring the monitoring of SR-IOV interface traffic
```

Comment:
Title is fine as is, but adding an alternative suggestion of "Configuring monitoring for SR-IOV interface traffic"

Reply:
Your suggestion is much cleaner, thank you!